<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFitFunctionsEditor</class>
 <widget class="QDialog" name="UserFitFunctionsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Fit Function Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QVisibleHandleSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="widList" native="true">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnNewF">
         <property name="text">
          <string>&amp;new fct.</string>
         </property>
         <property name="icon">
          <iconset resource="../lib/lib.qrc">
           <normaloff>:/lib/file_add.png</normaloff>:/lib/file_add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="btnDeleteF">
         <property name="text">
          <string>&amp;delete fct.</string>
         </property>
         <property name="icon">
          <iconset resource="../lib/lib.qrc">
           <normaloff>:/lib/file_delete.png</normaloff>:/lib/file_delete.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QListWidget" name="lstFunctions"/>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string> Fit Function </string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QVisibleHandleSplitter" name="splitter_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <layout class="QFormLayout" name="formLayout">
           <property name="fieldGrowthPolicy">
            <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ID:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="edtID"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>name/Description:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="edtName"/>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>short name:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="edtShortName"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>expression:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QFrame" name="horizontalFrame">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <widget class="QLineEdit" name="edtExpression"/>
              </item>
              <item row="0" column="1">
               <widget class="QToolButton" name="btnFunctionHelp">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../lib/lib.qrc">
                  <normaloff>:/lib/help/help.png</normaloff>:/lib/help/help.png</iconset>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="labFunctionOK">
                <property name="text">
                 <string/>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>file:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="labFilename">
             <property name="text">
              <string/>
             </property>
             <property name="scaledContents">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>parameters:</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0" rowspan="5">
              <widget class="QTableView" name="tvParams"/>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="btnAdd">
               <property name="text">
                <string>&amp;add</string>
               </property>
               <property name="icon">
                <iconset resource="../lib/lib.qrc">
                 <normaloff>:/lib/prop_add.png</normaloff>:/lib/prop_add.png</iconset>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QPushButton" name="btnDelete">
               <property name="text">
                <string>&amp;delete</string>
               </property>
               <property name="icon">
                <iconset resource="../lib/lib.qrc">
                 <normaloff>:/lib/prop_delete.png</normaloff>:/lib/prop_delete.png</iconset>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="3" column="1">
              <widget class="QPushButton" name="btnUp">
               <property name="text">
                <string>&amp;up</string>
               </property>
               <property name="icon">
                <iconset resource="../lib/lib.qrc">
                 <normaloff>:/lib/btn_up.png</normaloff>:/lib/btn_up.png</iconset>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QPushButton" name="btnDown">
               <property name="text">
                <string>&amp;down</string>
               </property>
               <property name="icon">
                <iconset resource="../lib/lib.qrc">
                 <normaloff>:/lib/btn_down.png</normaloff>:/lib/btn_down.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnSaveEdit">
           <property name="text">
            <string>&amp;Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnCancelEdit">
           <property name="text">
            <string>&amp;Cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVisibleHandleSplitter</class>
   <extends>QSplitter</extends>
   <header>qvisiblehandlesplitter.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>btnNewF</tabstop>
  <tabstop>btnDeleteF</tabstop>
  <tabstop>lstFunctions</tabstop>
  <tabstop>edtID</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>tvParams</tabstop>
  <tabstop>btnAdd</tabstop>
  <tabstop>btnDelete</tabstop>
  <tabstop>btnUp</tabstop>
  <tabstop>btnDown</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
  <include location="../lib/lib.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>UserFitFunctionsEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>UserFitFunctionsEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>helpRequested()</signal>
   <receiver>UserFitFunctionsEditor</receiver>
   <slot>showHelp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>451</x>
     <y>554</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>showHelp()</slot>
 </slots>
</ui>
